gprofiler/main.py

```diff
@@ -604,6 +624,8 @@ def main():
 
     setup_signals()
     reset_umask()
+    # assume we run in the root cgroup (when containerized, that's our view)
```

Comment:
can't we check it, the get_mode relies on arguments/environment variables, which the user might miss.

Reply:
Both `GPROFILER_IN_K8S` and `GPROFILER_IN_CONTAINER` are set by us, not by the user

Reply:
I know, but no better way to actually test it for sure?

Reply:
I agree that it's nicer if it is fully foolproof. What I think we'll add later on is that TODO I'd written before - moving ourselves into a sub-cgroup of `cpuacct` so it also works with executables. It will solve this issue as well

Reply:
ok